src/player.ts

```diff
@@ -405,4 +405,18 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
+   * @returns The currently selected video quality.
+   */
+  getVideoQuality = async (): Promise<number> => {
```

Comment:
This promise has wrong type, please use `VideoQuality`.
```suggestion
  getVideoQuality = async (): Promise<VideoQuality> => {
```

Reply:
Address in efb83e696e92f9db67998973f270e787b2da8d2c and 345c7fe21d44c7441167882021f901b390c29512